spec/shipcloud/carrier_spec.rb

```diff
@@ -70,7 +70,7 @@
 
   def stub_carriers_request
     allow(Shipcloud).to receive(:request)
-      .with(:get, 'carriers', {})
+      .with(:get, 'carriers', {}, api_key: nil)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.